bin/curriculum/remove_lesson_yml.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+# removes all remaining lesson fields from scripts.en.yml.
```

Comment:
Should we remove the one off script that was used to remove lesson descriptions now that its been run?